### PR TITLE
Add check that taxonomy term exists in services navigation

### LINKED ISSUE #157: WSOD on editing a services landing page if topic is empty
> Error: Call to a member function label() on null in Drupal\localgov_services_navigation\EntityChildRelationshipUi->childrenField() (line 179 of modules/contrib/localgov_services/modules/localgov_services_navigation/src/EntityChildRelationshipUi.php).

Debugging this line
````php
      $topics = [];
      if ($child->hasField('localgov_topic_classified')) {
        foreach ($child->localgov_topic_classified as $topic) {
          $topics[] = Html::escape($topic->entity->label());
        }
      }
````
So maybe if a topic was deleted there is an error here?

### PR DESCRIPTION
**Pull request by andybroomfield**
Fix #157 

Check that the entity term is of type TermInterface before trying to add it to the topics array.

### CODE REVIEW

**Comment by andybroomfield** [comment]
I need to look at where I add the test, but this should fix the issue,.

**Comment by finnlewis** [comment]
@andybroomfield still planning to add tests to this before merging.

**Review by ekes — APPROVED** [review]
I might have gone for assigning it and checking if it's empty; but as the field can only hold a Taxonomy Term it's much the same.

Tests make sense.

File: modules/localgov_services_navigation/src/EntityChildRelationshipUi.php [hunk 1]
```diff
@@ -176,7 +176,11 @@ protected function childrenField(NodeInterface $node) {
       $topics = [];
       if ($child->hasField('localgov_topic_classified')) {
         foreach ($child->localgov_topic_classified as $topic) {
-          $topics[] = Html::escape($topic->entity->label());
+          // Check that the taxonomy term has not been deleted.
+          // @see https://github.com/localgovdrupal/localgov_services/issues/157.
+          if ($topic->entity instanceof TermInterface) {
```
Comment (ekes) [inline modules/localgov_services_navigation/src/EntityChildRelationshipUi.php:181]: TermInterface doesn't have a use statement?

**Pull request merged**